app/views/candidate_interface/continuous_applications_choices/index.html.erb

```diff
@@ -6,25 +6,46 @@
       <%= t('page_titles.continuous_applications.your_applications') %>
     </h1>
 
-    <% unless @application_form_presenter.can_add_more_choices? %>
-      <p class="govuk-body">You cannot add any more applications.</p>
-      <p class="govuk-body">If one of your applications is unsuccessful, or you withdraw or remove it, you will be able to add another application.<p>
-          <p class="govuk-body">
-            <%= govuk_link_to 'Read how the application process works', candidate_interface_guidance_path %>
-          </p>
+    <% if CycleTimetable.today_is_between_apply_1_deadline_and_find_reopens? %>
+      <p class="govuk-body govuk-!-margin-top-6">Applications for courses starting in September <%= RecruitmentCycle.current_year %> are closed.</p>
```

Comment:
Appreciate we've now passed the A2 deadline but had we not then we would also be rendering this content for those still within the A2 window of applying right? Even though we intend to condense the two deadlines into a single date for the end of next cycle, until then I'm not too sure how I feel about the naming. Not necessarily a blocker but just interested in views! wdyt?

Reply:
You mean this right `today_is_between_apply_1_deadline_and_find_reopens?`

Doesn't the apply2 deadline fall within that time frame?

Reply:
it does but we would render this content immediately upon hitting the A1 deadline despite the A2 cohort having 2 weeks to still apply etc

Reply:
Discussed offline and decided to 🦵🏼 🍼 🛣️